cmake/SuperBuild.cmake

```diff
@@ -16,22 +16,26 @@ include(ExternalProject)
 
```

Comment:
Shall we consider other compilers?

 CMAKE_CACHE_ARGS
            -DCMAKE_CXX_COMPILER:FILEPATH=${CMAKE_CXX_COMPILER}
            -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}

Reply:
Added!